Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.py

```diff
@@ -364,15 +526,17 @@ def main():
             managed_identities_client_id=managed_identities_client_id,
         )
 
-        commands = {
+        commands: dict[str, Callable[[Client, dict], CommandResults | str]] = {
             'azure-log-analytics-execute-query': execute_query_command,
             'azure-log-analytics-list-saved-searches': list_saved_searches_command,
             'azure-log-analytics-get-saved-search-by-id': get_saved_search_by_id_command,
             'azure-log-analytics-create-or-update-saved-search': create_or_update_saved_search_command,
             'azure-log-analytics-delete-saved-search': delete_saved_search_command,
+            'azure-log-analytics-get-search-job': get_search_job_command,
+            'azure-log-analytics-delete-search-job': delete_search_job_command
```

Comment:
What about the run_search_job?

Reply:
cannot be inside the dictionary, because the first argument of the `polling function`
should be `args` and not `client` like the functions inside the dictionary

Reply:
Ok, but how do you run it? I see no reference for it in the main.

Reply:
line 561